3회차/남중식/2_암호생성기.py

```diff
@@ -1,3 +1,32 @@
 import sys
 
 sys.stdin = open("_암호생성기.txt")
+
+from collections import deque
+
+def swea_pw_gen():
+
+    for i in range(10):
+        case = int(input())
+        pw_list = list(map(int, input().split()))
+        pw_queue = deque(pw_list)
+        
+        while_checker = True
+        
+        while while_checker == True:
```

Comment:
```suggestion
        while while_checker:
```
`while_checker` 변수 자체를 조건으로 활용할 수도 있답니다 😄